.travis.yml

```diff
@@ -116,7 +128,7 @@ cache:
 before_install:
   - echo $TRAVIS_OS_NAME
   - |
-    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+    if [[ "$TRAVIS_OS_NAME" == "linux" && "$TRAVIS_CPU_ARCH" == "amd64" ]]; then
```

Comment:
Doesn't `ccache` work on ARM64? Should speed things up ....

If it really doesn't, can you add a comment here about what the problem is?